defaults/main.yml

```
@@ -1,8 +1,70 @@
-# SPDX-License-Identifier: MIT
 ---
-# Here is the right place to put the role's input variables.
-# This file also serves as a documentation for such a variables.
+sudo_rewrite_default_sudoers_file: True
```

```suggestion
sudo_rewrite_default_sudoers_file: true
```
Use lower spelling of booleans for consistency